src/mbgl/util/mapbox.cpp

```diff
@@ -11,15 +11,16 @@ namespace util {
 namespace mapbox {
 
 const std::string protocol = "mapbox://";
+const std::size_t protocol_length = protocol.length();
 const std::string baseURL = "https://api.mapbox.com/";
 
 bool isMapboxURL(const std::string& url) {
     return std::equal(protocol.begin(), protocol.end(), url.begin());
 }
 
-std::vector<std::string> getMapboxURLPathname(const std::string& url) {
+inline std::vector<std::string> getMapboxURLPathname(const std::string& url) {
```

Comment:
In this codebase, `inline` is used only where strictly necessary (i.e. non-template standalone definitions in header files).